### PR TITLE
Add procedures for tidal network example

### PR DESCRIPTION
**Pull request by zhangli-abcd**

### CODE REVIEW

File: draft-wqb-tvr-applicability.md [hunk 1]
```diff
@@ -506,7 +506,7 @@ Means to identify such dependency are needed.
 
 # TVR Use Cases with Code Example {#uc}
 
-## Energy Efficient
+## Overview of Tidal Network
```
Comment (QiufangMa) [inline draft-wqb-tvr-applicability.md:509]:
```suggestion
## Tidal Network
```

File: draft-wqb-tvr-applicability.md [hunk 2]
```diff
@@ -523,6 +523,140 @@ network nodes can be selectively disabled or enabled based on traffic patterns,
 thereby reducing the energy consumption of nodes during periods of low network
 traffic.
 
+## Architecture Example in Tidal Network
+
+As described in secion 3.1.2 of {{?I-D.ietf-tvr-requirements}},the locality of
```
Comment (QiufangMa) [inline draft-wqb-tvr-applicability.md:528]:
```suggestion
As described in {{Section 3.1.2 of ?I-D.ietf-tvr-requirements}}, the locality of
```

File: draft-wqb-tvr-applicability.md [hunk 3]
```diff
@@ -523,6 +523,140 @@ network nodes can be selectively disabled or enabled based on traffic patterns,
 thereby reducing the energy consumption of nodes during periods of low network
 traffic.
 
+## Architecture Example in Tidal Network
```
Comment (QiufangMa) [inline draft-wqb-tvr-applicability.md:526]:
```suggestion
## Architecture Example
```

File: draft-wqb-tvr-applicability.md [hunk 4]
```diff
@@ -523,6 +523,140 @@ network nodes can be selectively disabled or enabled based on traffic patterns,
...
+
+As described in secion 3.1.2 of {{?I-D.ietf-tvr-requirements}},the locality of
+schedule generation can be Centralized Generation or Distributed Generation. Both
```
Comment (QiufangMa) [inline draft-wqb-tvr-applicability.md:529]:
```suggestion
schedule generation can be centralized or distributed. Both
```

File: draft-wqb-tvr-applicability.md [hunk 5]
```diff
@@ -523,6 +523,140 @@ network nodes can be selectively disabled or enabled based on traffic patterns,
...
+
+### Architecture Example in Centralized Generation
+
+In the centralized schedule generation, the Schedule Service Requester in {{arch}}
+can be network administrator, and the Scheduled Service Responder can be a network
+controller or network management system. After generated schedules, the controller
+or management system need to determine whether to distribute these schedules based
+on the schedule Execution Locality defined in section 3.1.3 of{{?I-D.ietf-tvr-requirements}}.
+
+### Architecture Example in Distributed Generation
```
Comment (QiufangMa) [inline draft-wqb-tvr-applicability.md:540]:
```suggestion
### Distributed Generation
```

File: draft-wqb-tvr-applicability.md [hunk 6]
```diff
@@ -523,6 +523,140 @@ network nodes can be selectively disabled or enabled based on traffic patterns,
...
+
+In the distributed schedule generation，the Schedule Service Requester in {{arch}}
+can be network administrator, and the Scheduled Service Responders are the network
```
Comment (QiufangMa) [inline draft-wqb-tvr-applicability.md:543]:
```suggestion
can be network controller, and the Scheduled Service Responders are the network
```

File: draft-wqb-tvr-applicability.md [hunk 7]
```diff
@@ -523,6 +523,140 @@ network nodes can be selectively disabled or enabled based on traffic patterns,
...
+devices, the relationship of network administrator and network devices are shown in
```
Comment (QiufangMa) [inline draft-wqb-tvr-applicability.md:544]:
```suggestion
devices, the relationship between network controller and network devices is shown in
```

File: draft-wqb-tvr-applicability.md [hunk 8]
```diff
@@ -523,6 +523,140 @@ network nodes can be selectively disabled or enabled based on traffic patterns,
...
+{{arch-example}}. In this mode, the generation and execution of schedules are both
+on the same devices，so it does not involve the schedule distribution process.
```
Comment (QiufangMa) [inline draft-wqb-tvr-applicability.md:546]:
```suggestion
on the same devices, so it does not involve the schedule distribution process.
```

File: draft-wqb-tvr-applicability.md [hunk 9]
```diff
@@ -523,6 +523,140 @@ network nodes can be selectively disabled or enabled based on traffic patterns,
 thereby reducing the energy consumption of nodes during periods of low network
 traffic.
 
+## Architecture Example in Tidal Network
+
+As described in secion 3.1.2 of {{?I-D.ietf-tvr-requirements}},the locality of
+schedule generation can be Centralized Generation or Distributed Generation. Both
+of these two generation methods are applicable in Tidal Network.
+
+### Architecture Example in Centralized Generation
```
Comment (QiufangMa) [inline draft-wqb-tvr-applicability.md:532]:
```suggestion
### Centralized Generation
```

File: draft-wqb-tvr-applicability.md [hunk 10]
```diff
@@ -523,6 +523,140 @@ network nodes can be selectively disabled or enabled based on traffic patterns,
...
+(for example the traffic regularity in workday and weekend may totally different), the selection of tidal
+points should take full count of these factors. How to analyze the traffic patterns and determine the
+tidal points are outside the scope of this document.
+
+#### Estabilish Minimum and Peak Topology
+
+An algorithm is required to calculate the minimum and peak topology to service expected demand at different
+time slot. The calculation algorithm for the topology is outside the scope of this document.
+
+#### Generating Schedule
```
Comment (QiufangMa) [inline draft-wqb-tvr-applicability.md:589]: What is the relation between this section and above sections “Distributed Generation” and  “centralized Generation”？

**Review by QiufangMa — CHANGES_REQUESTED** [review]

File: draft-wqb-tvr-applicability.md (hunk 10 above)
Comment (zhangli-abcd) [reply draft-wqb-tvr-applicability.md:589]: Both "Distributed Generation" and "Centralized Generation" need the same Schedule Generation Prodecures, the only difference between "Distributed Generation" and "Centralized Generation"  is that "Centralized Generation" may need to distribute schedules to the execution devices.

**Review by QiufangMa — APPROVED** [review]

**Pull request merged**